python-shell/src/fishbowl/templates/base.py.j2

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2022 Crown Copyright
```

Comment:
Could jinja functionality be used in here to keep this up to date?

Reply:
Yes, I was thinking of doing it when it comes to 2023 so it can just say `2022-{{ currentYear }}` or something

Reply:
This could be introduced with other copyright date improvements - enabling spotless to work on Python. That can be done next year so we can include this.